front/components/app/RootLayout.tsx

```diff
@@ -14,7 +15,7 @@ function NextLinkWrapper({
   ariaLabel,
   onClick,
 }: {
-  href: string;
+  href: string | UrlObject;
```

Comment:
Require to align on the update shipped in https://github.com/dust-tt/dust/pull/4473 to better support NextJS Link.